src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -40,7 +40,7 @@
                         AutomationProperties.Name="{Binding ElementName=tbViewResults,Path=Text}"
                         AutomationProperties.AutomationId="{x:Static Properties:AutomationIDs.AutomatedChecksUIATreeButton}">
                     <StackPanel Orientation="Horizontal">
-                        <fabric:FabricIconControl GlyphName="DOM" FontSize="14" Foreground="{DynamicResource ResourceKey=ActiveBlueBrush}" Margin="0,0,4,0"/>
+                        <fabric:FabricIconControl GlyphName="DOM" FontSize="14" Foreground="{DynamicResource ResourceKey=ActiveBlueBrush}" Margin="0,0,4,0" ShowInControlView="False"/>
```

Comment:
![view results in UI automation tree - treeview icon](https://user-images.githubusercontent.com/7775527/64988877-40bcb300-d881-11e9-9959-d4eca5315d5f.png)